pkg/testidentification/ocp_variants.go

```diff
@@ -98,6 +99,20 @@ var (
 		"vsphere-ipi",
 		"vsphere-upi",
 	)
+
+	allPlatforms = sets.NewString(
```

Comment:
Should we add libvirt?

Reply:
Sure, we didn't have a libvirt variant but I just added one